dataland-frontend/src/components/generics/ViewMultipleDatasetsDisplayBase.vue

```diff
@@ -20,6 +20,11 @@
             <h2 class="mb-0" data-test="frameworkDataTableTitle">{{ humanizeString(dataType) }}</h2>
           </div>
           <div class="col-12">
+            <NewEuTaxonomyForNonFinancialsPanel
```

Comment:
This isn't the right location in the code for it but on the company search page New Eu Taxonomy for non-financial companies is still available in the drop down. It has to be removed.

![image](https://github.com/d-fine/Dataland/assets/35459207/61d5c9b6-bd6a-4bff-aec8-257f6195608b)

Reply:
Should be gone now.